README.md

```diff
@@ -14,21 +14,21 @@
 
 ## Disclaimer
 
-*Plutonium Addon: Automation* is **NOT** supported by any of the modules listed below. If you encounter an issue with your game, disable *Plutonium* and *Plutonium Addon: Automation* and try again.
+*Plutonium Addon: Automation* is **_not_** supported by any of the modules listed below. If you encounter an issue in your game, disable *Plutonium* and *Plutonium Addon: Automation* and try again. If an issue persists, consider raising an [issue](https://github.com/TheGiddyLimit/plutonium-addon-automation/issues) or asking about it in our [Discord server](https://discord.gg/5etools) before bringing it to the attention of other module authors.
 
-If an issue persists, consider asking about it in the [Issues section](https://github.com/TheGiddyLimit/plutonium-addon-automation/issues) or in our [Discord](https://discord.gg/5etools) before bringing it to the attention of other module authors.
-
-⚠️ Beware the rules of other spaces when discussing *Plutonium* and/or *Plutonium Addon: Automation* module outwith the above channels! ⚠️
+> [!WARNING]
+> Some places discourage discussion of *Plutonium* and *Plutonium Addon: Automation*. Be aware of any rules before mentioning these modules outside this repo and our [Discord server](https://discord.gg/5etools).
 
 ## How it works
 
-*Plutonium Addon: Automation* runs invisibly and mostly silently.
+*Plutonium Addon: Automation* runs invisibly and (mostly) silently.
 
-While the module is activated in your world, each time Plutonium would import a document, it will check if any automation data is available for that document. If so, it will import the document with the additional automation data included. If not, it will import the document as Plutonium standard (although sometimes slightly tweaked to better enable the module dependencies outlined below).
+While the module is activated in your world, each time Plutonium would import a document, it will check if any automation data is available for that document. If so, it will import the document with the additional automation data included. If not, it will import the document standardly for Plutonium (although sometimes slightly and non-destructively tweaked for better compatibility with the modules below).
```

Comment:
```suggestion
While the module is activated in your world, each time Plutonium would import a document, it will check if any automation data is available for that document. If so, it will import the document with the additional automation data included. If not, it will import the document as if *Plutonium Addon: Automation* was not enabled (although sometimes slightly and non-destructively tweaked for better compatibility with the modules below).
```

you've somehow managed to make my clunky wording even worse

Impressive!

Reply:
I thought I made it less clunky :coolcry:

Accepted your suggestion but with a 'was' to 'were' nitpick 😏

Reply:
oops ye ty

you cannot possibly convince me that any deployment of the word `standardly` is "less clunky"